sql/drop.go

```diff
@@ -95,8 +94,7 @@ func (p *planner) DropDatabase(n *parser.DropDatabase) (planNode, error) {
 //   Notes: postgres allows only the index owner to DROP an index.
 //          mysql requires the INDEX privilege on the table.
 func (p *planner) DropIndex(n *parser.DropIndex) (planNode, error) {
-	var b client.Batch
```

Comment:
it's in a separate commit that makes it consistent in `sql/` - i do prefer implicit zero, but some of the batch definitions are adjacent to to non-zero definitions, and I chose the visual consistency